requirements.txt

```diff
@@ -3,4 +3,4 @@ pytest==3.1.2
 pytest-cov==2.5.1
 responses==0.5.1
 coveralls
-glom==18.3.1
+glom==18.3.1
```

Comment:
need to add deprecated to the requirements (you'll get an import error otherwise)